### ISSUE TITLE
LSC-Bell 8S refuse to work without inet

### ISSUE THREAD

**Issue by jonesMeUp** [jasonacox/tinytuya#161]
thx for this project!
i am using fhem and as this project is platform independant, i will have a look at the progress here ;)

i have a LSC-Bell 8S(AKV300_8M) where i do some testing right now.
and as soon as i get it freed from the internet, i will make a doorbellDevice for tinytuya.
(all the dps are known already, tinytuya.getdps() works very well)

i have a video stream on rtsp://192.168.178.25:8554/Streaming/Channels/101

`d = tinytuya.OutletDevice('xxxxxxxxxxxxxxxxx', '192.168.178.25', 'xxxxxxxxxxxxxxxx', 'device22')`
`d.set_version(3.3)`
`d.set_socketPersistent(True)`
`while(True):`
`    print('Received Payload: %r' % d.receive())`
`    d.send(d.generate_payload(tinytuya.HEART_BEAT))`  

_this is what i get when ip is not blocked on router (both events happens on bell pressed):_
  `Received Payload: {'dps': {'136': '1657817163'}, 't': 1657817163} ` # bell was pressed
  `Received Payload: {'dps': {'154': 'aHR0cHM6Ly90    ...'}, 't': 1657817164}` # photo from visitor

but as soon as i block the ip in my router, the device refuses to give me the status update and video stream from above.
commands like 'basic_device_volume' are still working.

switching power off and on is no problem "only" missing internet is.
smartlife app running or not, after powering, up makes no difference.

i played some time with the updatedps() function, but without any success:
- tried all used dps from the doorbell
- tried lower dps 18,19,20,21,22,...
- tried with and without nowait

**it makes no sense to run the doorbell in my area** (inet is not stable and there are times when i have no inet for > 1day)
so i could not hear the doorbell for some days in every month.
now i ran out of ideas and really could use some help on this.. thx

**Comment by uzlonewolf** [comment]
It seems that camera really does not like being offline.  There is a hack to make it work if you have firmware version 4.x at https://github.com/guino/BazzDoorbell/wiki/Can-I-use-the-device-OFFLINE-(disconnected-from-tuya-servers)-%3F however it does not work with newer 5.x firmware.

**Comment by jonesMeUp** [comment]
thx uzlonewolf for the fast answer.
though i found guino/BazzDoorbell i didn't find this offline instruction. 
I will have a look into it and i hope it will work for me.

if it will work, i will make a new issue with the doorbellDevice for tinytuya.
don't know if anybody needs it, but i always want to give something back.

**Comment by jasonacox** [comment]
Thanks for opening this issue @jonesMeUp and please post any additional discovery.  

If you do want to submit a device module for a `DoorbellDevice`, please see the great template that @uzlonewolf set up here: https://github.com/jasonacox/tinytuya/tree/master/tinytuya/Contrib
